docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  redis:
+    image: redis:5
+    ports:
+      - "6379:6379"
+  arango:
+    image: arangodb:3.2
+    ports:
+      - "8529:8529"
+    environment:
+      - ARANGO_NO_AUTH=1
```

Comment:
:rhinoceros: :old_key: :closed_lock_with_key:

Reply:
Obviously not a production setup :see_no_evil: